ras_party/views/respondent_view.py

```diff
@@ -77,6 +77,18 @@ def _validate_get_respondent_params(ids, first_name, last_name, email):
                 raise BadRequest(f"'{party_id}' is not a valid UUID format for property 'id'")
 
 
+@respondent_view.route("/respondents/party_id/<party_id>", methods=["GET"])
+def get_respondent_by_party_id(party_id: UUID) -> Response:
+    if not is_valid_uuid4(party_id):
+        return make_response("party_id is not UUID", 400)
+
+    respondent = respondent_controller.get_respondent_by_party_id(party_id)
+    if respondent:
```

Comment:
I only want to deal with the object until it gets returned, why the conversion here.